src/runtime/audience-activity-listener.js

```diff
@@ -75,24 +74,26 @@ export class AudienceActivityEventListener {
    * @private
    */
   handleClientEvent_(event) {
-    if (
-      this.storage_.get(Constants.USER_TOKEN) &&
-      event.eventType &&
-      audienceActivityLoggingEvents.has(event.eventType)
-    ) {
-      const pubId = encodeURIComponent(
-        this.deps_.pageConfig().getPublicationId()
-      );
-      const audienceActivityClientLogsRequest = this.createLogRequest_(event);
-      const url = serviceUrl(
-        '/publication/' +
-          pubId +
-          '/audienceactivitylogs' +
-          '&sut=' +
-          Constants.USER_TOKEN
-      );
-      this.fetcher_.sendBeacon(url, audienceActivityClientLogsRequest);
-    }
+    this.storage_.get(Constants.USER_TOKEN, true).then((swgUserToken) => {
```

Comment:
For a slight performance improvement, I'd recommend checking event type out of the promise, so that we don't even have to enter the promise statement unnecessary for the majority of event types that are not in our allowlist.
```
if (event type not in audienceActivityLoggingEvents) {
  return;
}
this.storage_.get()...
```